src/test/scala/uk/gov/nationalarchives/FileProcessorTest.scala

```diff
@@ -42,7 +42,7 @@ class FileProcessorTest extends AnyFlatSpec with MockitoSugar with TableDrivenPr
 
   val metadataJson: String =
     s"""{"parameters":{"TDR": {"Document-Checksum-sha256": "abcde", "Source-Organization": "test-organisation",
-       | "Internal-Sender-Identifier": "test-identifier","Consignment-Export-Datetime": "2023-10-31T13:40:54Z"},
+       | "Internal-Sender-Identifier": "test-identifier","Consignment-Export-Datetime": "2023-10-31T13:40:54Z", "UUID": "24190792-a2e5-43a0-a9e9-6a0580905d90"},
```

Comment:
1. I'm not able to add a comment on line 53 but shouldn't UUID also be added to the `tdrParams` Map, right below?
    a. And then on line 280, you could just refer to the `tdrParams` Map?
3. Do you think it's worth doing something like this so that they are always in sync?

``` {"TDR": ${upickle.default.write(tdrParams)}, ...}```

Reply:
Ah yeah good point. I've changed it. This is using circe now instead of upickle so I've used that.

Reply:
Regarding point "a." above, did you leave this out because the UUID on line 280 has to be random?

Reply:
No, I left it out because I missed it :smile:  I think it makes more sense like it is now.